### ISSUE TITLE
Got a panick when a backend is remove with sozuctl

### ISSUE THREAD

**Issue by NotBad4U** [sozu-proxy/sozu#488]
When I try to remove a TCP backend from the `bin/config.toml` example with `sozuctl`: 

```bash
./target/debug/sozuctl -c bin/config.toml backend remove
--backend-id TcpTest-0
--id TcpTest
--ip 127.0.0.1
--port 4001
Command timeout. The proxy didn't send answer
```
The main process panick 
> main' panicked at 'attempt to subtract with overflow'

This is the full logs and the problem seem's to come from [orders.rs:504:34](https://github.com/sozu-proxy/sozu/blob/b177cfef74e4d898422453268d380f5b1cb66157/bin/src/command/orders.rs#L504)
```bash
2018-07-02T09:55:50Z 195573828336904 20573 MASTER DEBUG	sozu::command::orders	proxyconfig client order RemoveBackend(RemoveBackend { app_id: "TcpTest", backend_id: "TcpTest-0", ip_address: "127.0.0.1", port: 4001 })
thread 'main' panicked at 'attempt to subtract with overflow', bin/src/command/orders.rs:504:34
note: Run with `RUST_BACKTRACE=1` for a backtrace.
2018-07-02T09:55:50Z 195573828620517 20575 WRK-01 ERROR	command channel was closed
2018-07-02T09:55:50Z 195573828623590 20574 WRK-00 ERROR	command channel was closed
```

NOTE: It's easy to reproduce

**Comment by Geal** [comment]
was there already a backend in the configuration?

**Comment by NotBad4U** [comment]
Yeah, I used the configuration in `/bin/config.toml`

**Comment by Geal** [comment]
fixed in 2510e1f
